public/app/core/components/TraceToLogsSettings.tsx

```diff
@@ -6,12 +6,14 @@ import {
   updateDatasourcePluginJsonDataOption,
 } from '@grafana/data';
 import { DataSourcePicker } from '@grafana/runtime';
-import { InlineField, InlineFieldRow, TagsInput, useStyles } from '@grafana/ui';
+import { InlineField, InlineFieldRow, Input, TagsInput, useStyles } from '@grafana/ui';
 import React from 'react';
 
 export interface TraceToLogsOptions {
   datasourceUid?: string;
   tags?: string[];
+  spanStartTimeShift?: number;
+  spanEndTimeShift?: number;
```

Comment:
Can we instead use the same code that is used for other time inputs? That can accept strings like `1s` or `5m`? Probably with defaulting to milliseconds if there is no unit. To align it with other time inputs.

Reply:
Good idea!